lib/msf/core/exploit/dns/enumeration.rb

```diff
@@ -61,8 +61,8 @@ def dns_axfr(domain)
         print_error("Query #{domain} DNS AXFR - unknown exception: #{e}")
       end
       next if zone.blank?
-      records << zone
-      print_good("#{domain} Zone Transfer: #{zone}")
+      records += zone
+      print_good("#{domain} Zone Transfer: #{zone.map(&:inspect).join}")
```

Comment:
Since the zone starts with `;;`, I noticed that it looks a little awkward this way:
```
[+] zonetransfer.me Zone Transfer: ;; Answer received from 81.4.108.41:53 (1983 bytes)
;;
;; HEADER SECTION
```

I think it might look a little better if the zone info started on it's own new line:
```suggestion
      print_good("#{domain} Zone Transfer:")
      print_line(zone.map(&:inspect).join)
```

Which would change the output to:
```
[+] zonetransfer.me Zone Transfer:
;; Answer received from 81.4.108.41:53 (1983 bytes)
;;
;; HEADER SECTION
```

Reply:
Yeah that one irked me as well, I'll get that fixed :)